src/features/island/hud/components/buffs/CollectionBuffsList.tsx

```diff
@@ -0,0 +1,90 @@
+import chest from "assets/icons/chest.png";
+import { Label } from "components/ui/Label";
+import { OuterPanel } from "components/ui/Panel";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { getEntries } from "features/game/types/craftables";
+import { Collectibles } from "features/game/types/game";
+import { OPEN_SEA_ITEMS } from "metadata/metadata";
```

Comment:
I found this data source that I used to obtain information about the collectibles. Is there a better way?

Reply:
Hey @joaofernandez first of all, amazing work you've done here!
maybe you can check "src\features\game\types\collectibles.ts"?